files/build_templates/docker_image_ctl.j2

```diff
@@ -56,6 +56,7 @@ start() {
         --log-opt max-size=2M --log-opt max-file=5 \
 {%- endif %}
         -v /var/run/redis:/var/run/redis:rw \
+        -v /etc/timezone:/etc/timezone:rw \
```

Comment:
do you need to configure as rw? can you mount as ro? since you want the timezone in docker to be in sync with baseimage, then you do not really want to change the timezone in the baseimage, right?

Reply:
I tried ro, but dpkg-reconfigure -f noninteractive tzdata need write permission, or it will not update ok. this command will not change the file content.